ndcube/tests/test_global_coords.py

```diff
@@ -0,0 +1,85 @@
+import astropy.units as u
+import pytest
+
+from ndcube.global_coords import GlobalCoords
+
+
+@pytest.fixture
+def global_coords(ndcube_3d_ln_lt_l):
+    return GlobalCoords(ndcube_3d_ln_lt_l)
+
+
+def test_adding_global_coords(global_coords):
+    coord1 = 1 * u.m
+    coord2 = 2 * u.s
+    global_coords.add('name1', 'physical_type1', coord1)
+    global_coords.add('name2', 'physical_type2', coord2)
+    assert global_coords.names == ('name1', 'name2')
+    assert global_coords.physical_types == ('physical_type1', 'physical_type2')
+
+
+def test_removing_global_coords(global_coords):
+    coord1 = 1 * u.m
+    coord2 = 2 * u.s
+    global_coords.add('name1', 'physical_type1', coord1)
+    global_coords.add('name2', 'physical_type2', coord2)
+    global_coords.remove('name2')
+    assert len(global_coords) == 1
+    assert global_coords.names == ('name1',)
+    assert global_coords.physical_types == ('physical_type1',)
+
+
+def test_iterating(global_coords):
```

Comment:
See comment in thread above.  As said there, this is not the behaviour we want.

Reply:
Could you elaborate a bit on the expected behavior? I think I am a bit confused from the discussion previously.

Reply:
I tried `.items()` already but that would yield some other behavior.

Reply:
So resorted to using `enumerate()` in the `for` loop...